packages/ubus-lime-utils/files/usr/libexec/rpcd/lime-utils

```diff
@@ -189,13 +189,26 @@ local function change_config (msg)
     end
 end
 
+local function set_wifi(msg)
+    if msg ~= nil then
+        for key, value in pairs(msg) do
+            conn:call("uci", "set", {config="lime", section="wifi", values= { key = value }})
```

Comment:
the only reason this function is called set_wifi is because there is a "wifi" value in the section.
I guess it would be much useful if it is called lime_bulk_set(section, values) or something.

Why do we need this?